jipp-pdl/src/main/kotlin/com/hp/jipp/pdl/pwg/PwgSettings.kt

```diff
@@ -24,7 +24,13 @@ data class PwgSettings @JvmOverloads constructor(
     val sheetBack: String = PwgRasterDocumentSheetBack.normal,
 
     /** The orientation used when printing the current job. */
-    val orientation: Orientation = Orientation.portrait
+    val orientation: Orientation = Orientation.portrait,
+
+    /**
+     * True to add blank page padding for two-sided odd-page-count jobs, or false the destination knows how to pad
+     * them (for example it supports `job-pages-per-set`).
+     */
+    val allowPadding: Boolean = false,
```

Comment:
this needs to be true, else the issue will not be solved.
if any test is using `job-pages-per-set` then MCT can send false for this parameter, but the default has to be true.

Reply:
The caller can specify whichever they want. But perhaps true is a safer default, so I'll make the change.